openedx/adg/lms/user_authn/login_form.py

```diff
@@ -0,0 +1,55 @@
+"""
+Login form and views
+"""
+
+from django.urls import reverse
+from django.utils.translation import ugettext as _
+
+from openedx.core.djangoapps.user_api import accounts
+from openedx.core.djangoapps.user_api.helpers import FormDescription
+from openedx.core.djangoapps.user_authn.views.login_form import _apply_third_party_auth_overrides
+from util.password_policy_validators import DEFAULT_MAX_PASSWORD_LENGTH
+
+
+def get_login_session_form(request):
+    """
+    Return a description of the login form.
+
+    This decouples clients from the API definition:
+    if the API decides to modify the form, clients won't need
+    to be updated.
+
+    See `user_api.helpers.FormDescription` for examples
+    of the JSON-encoded form description.
+
+    Returns:
+        HttpResponse
+
```

Comment:
Can we please change it to elaborate, that we are overriding the core method, may be way add path of the core method as well.